packages/paste-core/components/typography/src/types.ts

```diff
@@ -0,0 +1,5 @@
+export interface ParagraphProps {
```

Comment:
Just playing out this file structure in my mind, the eventuality would be something like

```
export {Paragraph} from './Paragraph';
export {List} from './List';
export {Heading} from './Heading';
export {ParagraphProps, ListProps, HeadingProps} from './types';
```

Two thoughts:
1- I kind of wonder (since these components are so minimal) if we should move the types into the same file? 
2- We can actually name them the same thing as the component.

So it becomes
```
export {Paragraph} from './Paragraph';
export {List} from './List';
export {Heading} from './Heading';
```

You are allowed to name an interface and a value the same thing, and Typescript knows how to infer which one you mean in code.

```
interface Paragraph {}
const Paragraph: React.FC<Paragraph> = () => {};
export {Paragraph}
```
should work.

https://stackoverflow.com/questions/49798156/typescript-variable-name-as-interface-type

Reply:
Love this 👍